<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kategorie</class>
 <widget class="QDialog" name="Kategorie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kategorie bearbeiten</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Hauptkategorie</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Unterkategorie</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="tbHaupt">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="4">
    <widget class="QTableWidget" name="twHaupt">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="tbHaupt2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="5" rowspan="4">
    <widget class="QTableWidget" name="twUnter">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QLineEdit" name="tbUnter">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <spacer name="verticalSpacer_2">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>118</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="btAddHaupt">
     <property name="text">
      <string>Hinzuf端gen</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Lagerort</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Marke</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLineEdit" name="tbLager">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="2" rowspan="3">
    <widget class="QTableWidget" name="twLager">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QLineEdit" name="tbMarke">
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="5" rowspan="3">
    <widget class="QTableWidget" name="twMarke">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer_4">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="4">
    <spacer name="verticalSpacer_3">
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>88</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QPushButton" name="btAddLager">
     <property name="text">
      <string>Hinzuf端gen</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPushButton" name="btAddUnter">
     <property name="text">
      <string>Hinzuf端gen</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QPushButton" name="btAddMarke">
     <property name="text">
      <string>Hinzuf端gen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
